lib/services/ngsi/subscriptionService.js

```diff
@@ -126,11 +126,12 @@ function subscribe(device, triggers, content, callback) {
         store = false;
     }
 
-    if (device.cbHost) {
-        options.uri = 'http://' + device.cbHost + '/v1/subscribeContext';
+    if (device.cbHost && device.cbHost.indexOf("://") !== -1) {
```

Comment:
Should be the NGSIv2 code (using `/v2/subscription`) adapted in a similar mannter? What do you think @dcalvoalonso ?

In positive case, I'll hold this PR until NGSIv2 code gets integrated.

Reply:
The changes are minor. I can do it easily in NGSIv2 so I propose you not to wait so that @chicco785  can go on with his work! ;)

Reply:
Ok. We can keep #593 opened while that missing piece is pending, after merging this PR.